_includes/pbc/all/install-features/202304.0/install-push-notification-feature.md

```diff
@@ -0,0 +1,698 @@
+| DeleteExpiredPushNotificationSubscriptionConsole | Delete expired push notification subscriptions from Persistence. |               | Spryker\Zed\PushNotification\Communication\Console |
+
+**src/Pyz/Zed/Console/ConsoleDependencyProvider.php**
+
+```php
+<?php
+
+namespace Pyz\Zed\Console;
+
+use Spryker\Zed\Console\ConsoleDependencyProvider as SprykerConsoleDependencyProvider;
+use Spryker\Zed\PushNotification\Communication\Console\DeleteExpiredPushNotificationSubscriptionConsole;
+use Spryker\Zed\PushNotification\Communication\Console\SendPushNotificationConsole;
+
+class ConsoleDependencyProvider extends SprykerConsoleDependencyProvider
+{
+     /**
+     * @param \Spryker\Zed\Kernel\Container $container
+     *
+     * @return array<\Symfony\Component\Console\Command\Command>
+     */
+    protected function getConsoleCommands(Container $container): array
+    { 
+        return [
+            new DeleteExpiredPushNotificationSubscriptionConsole(),
+            new SendPushNotificationConsole(),
+        ];
+    }
+}
+
+```
+
+4. Enable the Backend API by registering the plugin:
+
+| PLUGIN                                     | SPECIFICATION                                             | PREREQUISITES | NAMESPACE                                                       |
+|--------------------------------------------|-----------------------------------------------------------|---------------|-----------------------------------------------------------------|
+| PushNotificationSubscriptionResourcePlugin | Registers the `push-notification-subscriptions` resource. |               | Spryker\Glue\PushNotificationsBackendApi\Plugin\GlueApplication |
+
+**src/Pyz/Glue/GlueBackendApiApplication/GlueBackendApiApplicationDependencyProvider.php**
+
+```php
+<?php
+
+namespace Pyz\Glue\GlueBackendApiApplication;
+
+use Spryker\Glue\GlueBackendApiApplication\GlueBackendApiApplicationDependencyProvider as SprykerGlueBackendApiApplicationDependencyProvider;
+use Spryker\Glue\PushNotificationsBackendApi\Plugin\GlueApplication\PushNotificationSubscriptionResourcePlugin;
+
+class GlueBackendApiApplicationDependencyProvider extends SprykerGlueBackendApiApplicationDependencyProvider
+{
+    /**
+     * @return array<\Spryker\Glue\GlueApplicationExtension\Dependency\Plugin\ResourceInterface>
+     */
+    protected function getResourcePlugins(): array
+    {
+        return [
+            new PushNotificationSubscriptionResourcePlugin(),
+        ];
+    }
+}
+
+```
+
+{% info_block warningBox "Verification" %}
+
+Ensure that the plugins work correctly:
+1. To test the functionality, create a simple single-page demo application.
+2. Generate VAPID keys through the online generator https://vapidkeys.com/.
+3. Create a directory for the demo application: `mkdir push_notification_spa`.
+4. In the `push_notification_spa` directory, create the following files:
+
+**.../push_notification_spa/index.html**
```

Comment:
Shouldn't we provide a full path? Is it enough to have only part of it?

Reply:
The full path we don't know because the customers have to create the folder on their own.